CONTRIBUTING.md

```diff
@@ -267,28 +267,22 @@ to install any additional tools.
 
 CPU Docker: 
 ```
-docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:2.1.0-custom-op-ubuntu16
+docker run --rm -it -v ${PWD}:/addons -w /addons tfaddons/dev_container:latest
 ```
 
 GPU Docker: 
 ```
-docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:2.1.0-custom-op-gpu-ubuntu16
-```
-
-Configure:
-```
-python3 -m pip install tensorflow==2.2.0
-python3 ./configure.py  # Links project with TensorFlow dependency
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tfaddons/dev_container:latest
 ```
 
 Install in editable mode
 ```
-python3 -m pip install -e .
-python3 -m pip install -r tools/install_deps/pytest.txt
+python -m pip install -e .
 ```
 
 Compile the custom ops
 ```
+export TF_NEED_CUDA=1 # If GPU is to be used
```

Comment:
This seems that is set by default to 1 in the image/container. 
So if I follow CPU custom_op documentation it build cuda files also without exporting this env.

Reply:
As we are using the `custom_ops` GPU image version as our baseline it is inherited by https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.custom_op_gpu#L21

Reply:
Resolved since using the cpu custom-op image as base.